code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -257,16 +257,16 @@ datum/gear/suit/colorvest
 	gear_tweaks += new /datum/gear_tweak/path(secjacket)
 
 /datum/gear/suit/secjackethos
-	display_name = "security jacket selection (Head of Security)"
-	path = /obj/item/clothing/suit/storage/security/hos
-	allowed_roles = list("Head of Security")
+	display_name = "security jacket selection (Chief of Security)"
```

Comment:
```suggestion
	display_name = "security jacket selection (chief of security)"
```

Reply:
This is not correct. This shows, e.g. in the loadout, in brackets who this stuff is allowed to spawn on. THERE it is capitalized (as seen on the removed lines)